gen/builders/generator.go

```diff
@@ -403,3 +418,21 @@ func ensureDirectory(path string) error {
 	}
 	return nil
 }
+
+// removePreviouslyBroken will remove a previously broken vector file if it
+// became fixed & removes a previously working vector file if it became broken.
+func removePreviouslyBroken(dir string, group string, item *VectorDef) error {
```

Comment:
Nitpick: I find this name a bit confusing, since this is also going to remove a previously working vector file if it became broken.

Suggestion: rename to `removePreviousVector` (or `removePrevious`) since it removes both broken and working previous vector files.